runtime/src/main/java/io/quarkus/reactivemessaging/http/runtime/ReactiveHttpHandlerBean.java

```diff
@@ -59,7 +59,11 @@ protected void handleRequest(RoutingContext event, MultiEmitter<? super HttpMess
         } else if (guard.prepareToEmit()) {
             try {
                 HttpMessage<Buffer> message = new HttpMessage<>(event.getBody(), new IncomingHttpMetadata(event.request()),
-                        () -> event.response().setStatusCode(202).end(),
+                        () -> {
+                            if (!event.response().ended()) {
```

Comment:
when this will be ended?

Reply:
it ends on first message acknowledge from subscribers.

Reply:
does it mean that the result of the first subscriber "wins"? In other words, if the result for the first one is a failure, 500 will  be returned, otherwise 202?

Reply:
yes, that is how it behaves now.

Reply:
it would be great if we got something consistent. And maybe another config option that would say what the behavior should be?

@cescoffier do you have an opinion?

Reply:
What I'm thinking is a user may have e.g. one consumer for logging and one for the real job. And if the logging one succeeded and the "proper" one failed, a user may have an illusion that it works even if the real job is not done

Reply:
Unfortunately, broadcast does not handle acknowledgment well (we have it on the roadmap for a long time).  At the moment it's a _race_: the first one wins.  If the order is important, that are other approaches (but you would need to write some code in the application).

Reply:
so IIUC there's nothing we can do here. I'm merging this